### ISSUE TITLE
UnknownApiNameOrVersion: name: webmasters  version: v3

### ISSUE THREAD

**Issue by hecklerponics** [joshcarty/google-searchconsole#49]
Currently troubleshooting an issue where the library no longer authenticates; there's several credentials I'm using to test this and it seems to be across all of them (_which I originally thought was a refresh token issue_), here's the relevant stacktrace: 

```
UnknownApiNameOrVersion                   Traceback (most recent call last)
<ipython-input-3-0a688b09596c> in <module>
      1 account = gsc.authenticate(client_config='gsc_auth_files/lsg3_auth.json', 
----> 2                            credentials='gsc_auth_files/lsg3_creds.json')

E:\python\anaconda\envs\gsc_covid\lib\site-packages\searchconsole\auth.py in authenticate(client_config, credentials, serialize, flow)
     97         version='v3',
     98         credentials=credentials,
---> 99         cache_discovery=False,
    100     )
    101 

E:\python\anaconda\envs\gsc_covid\lib\site-packages\googleapiclient\_helpers.py in positional_wrapper(*args, **kwargs)
    132                 elif positional_parameters_enforcement == POSITIONAL_WARNING:
    133                     logger.warning(message)
--> 134             return wrapped(*args, **kwargs)
    135 
    136         return positional_wrapper

E:\python\anaconda\envs\gsc_covid\lib\site-packages\googleapiclient\discovery.py in build(serviceName, version, http, discoveryServiceUrl, developerKey, model, requestBuilder, credentials, cache_discovery, cache, client_options, adc_cert_path, adc_key_path, num_retries)
    268                 raise e
    269 
--> 270     raise UnknownApiNameOrVersion("name: %s  version: %s" % (serviceName, version))
    271 
    272 

UnknownApiNameOrVersion: name: webmasters  version: v3
```
I'm also unable to get the redirect to work when authenticating via oAuth after signing in, it either refuses connection or gives `The authentication flow has completed. You may close this window.` and then never finishes executing and I have to kill the Kernel. 

Curious if anyone has any insight as to what is up here?

**Comment by joshcarty** [comment]
Hey - yeah should be an easy fix:

tl;dr: upgrade to the latest version of the package with:

`$ pip install -U git+https://github.com/joshcarty/google-searchconsole`

Long answer: last year Google [announced](https://developers.google.com/search/blog/2020/12/search-console-api-updates) that they were upgrading the underlying API from "webmasters v3" to "searchconsole v1". It looks like they have finally deprecated the old "webmasters" endpoint and it can no longer be found.

Let me know if that works for you.

**Comment by hecklerponics** [comment]
Looks to have resolved it, thanks @joshcarty
